realm/realm-library/src/main/java/io/realm/internal/OsSchemaInfo.java

```diff
@@ -16,8 +16,6 @@
 
 package io.realm.internal;
 
-import java.util.List;
```

Comment:
If List was imported as `java.util.List` here, can `Collection` be imported here by `import java.util.Collection;`? Since there is no other Collection in this file, it seems not affecting `OsSchemaInfo`'s namespace.